starport/templates/testutil/register.go

```diff
@@ -40,11 +43,20 @@ func Register(ctx *plush.Context, gen *genny.Generator, appPath string) error {
 		return err
 	}
 
-	// if yes, only box the sample folder
+	// if yes, only box the sample and normalize folder
```

Comment:
What does it mean normalizing the folder?

Reply:
weird, I think I put the comment in the wrong place, sorry 😁